addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -221,6 +235,14 @@ function clickOnText(snippet, element, position = "bottom") {
     };
 }
 
+function clickOnWebsite(position = "bottom") {
+    return {
+        trigger: ".o_app[data-menu-xmlid='website.menu_website_configuration']",
+        content: Markup(_t("<b>Click Website Icon</b> to Open Website.")),
+        position: position,
+    };
+}
```

Comment:
I'm surprised. These are the website tours. I would have expected them to already have some way to open the website. Why do we need a new one ?